templates/download/nvidia-jetson/index.html

```diff
@@ -0,0 +1,80 @@
+{% extends "download/_base_download.html" %}
+
+{% block title %}Install Ubuntu on NVIDIA Jetson{% endblock %}
+
+{% block meta_description %}Use Ubuntu on NVIDIA Jetson for the familiar developer experience and an accelerated path to production{% endblock meta_description %}
+
+{% block meta_copydoc%}https://docs.google.com/document/d/1DHLqBbo6xIHDp-Uc8ngo1t5-GJgeTdTVwBvZRIKyZ0Y/edit{% endblock meta_copydoc %}
+
+{% block content %}
+
+<section class="p-strip is-shallow u-no-padding--bottom">
+  <div class="row--25-75 p-section">
+    <div class="col u-hide--small u-hide--medium">
+      <div class="p-image-wrapper">
+        {{ image (
+          url="https://assets.ubuntu.com/v1/cb6924a9-Nvidia_image_logo.svg",
+          alt="Nvidia",
+          width="100",
+          height="74",
+          hi_def=True,
+          loading="auto"
+          ) | safe
+        }}
+      </div>
+    </div>
+    <div class="col">
+      <h1>Install Ubuntu on NVIDIA Jetson</h1>
+      <p>Run Ubuntu on your NVIDIA&reg; Jetson&trade; Orin platforms to create breakthrough AI solutions across all industries, being available for all the enthusiasts for hands-on AI learning and making amazing projects and delivering commercial solutions.</p>
+      <p>Pick the OS image to match your hardware, flash it onto SD/microSD card, load it onto your board and away you go.</p>
+      <p>All images are 64-bit builds of Ubuntu.</p>
+      <hr>
+      <p>
+        <a href="/internet-of-things/nvidia/contact-us" class="p-button--positive">Get in touch</a>
+      </p>
+    </div>
+  </div>
+</section>
+
+<section class="p-strip is-deep u-no-padding--top">
```

Comment:
```suggestion
<section class="p-section--deep">
```
The only difference (that I know of) between strips and sections is that the former has top padding and the other doesn't so this would do the same thing with less classes. Works either way though, totally up to you

Reply:
You are right! I actually didn't know there was a deep variety of `p-section`, thanks!